### ISSUE TITLE
Missing files on first-time pip install of galaxy-app package, which resolves on upgrade or reinstall

### ISSUE THREAD

**Issue by nuwang** [galaxyproject/galaxy#14021]
**Describe the bug**
Installing the galaxy-app package via pip results in a strange situation where resource files appear to be missing on initial install, but get installed if the same package is reinstalled or upgraded. Is this a pip bug?

**To Reproduce**

***Method 1 (install, uninstall, reinstall):***
Create a virtualenv and
1. ```
   pip install galaxy-app@git+https://github.com/galaxyproject/galaxy.git@release_21.09#subdirectory=packages/app
   ```
2. ```
    find . -name "exp_history_to_archive.xml"
    ```
   `No files found.`
3. ```
    pip uninstall galaxy-app
    ```
4. ```
    pip install galaxy-app@git+https://github.com/galaxyproject/galaxy.git@release_21.09#subdirectory=packages/app
    ```
5. ```
    find . -name "exp_history_to_archive.xml"
    ```
    `./venv/lib/python3.8/site-packages/galaxy/tools/imp_exp/exp_history_to_archive.xml`


***Method 2 (install, upgrade):***
Create a virtualenv and

1. ```
    pip install galaxy-app@git+https://github.com/galaxyproject/galaxy.git@release_21.09#subdirectory=packages/app
    ```
2. ```
    find . -name "exp_history_to_archive.xml"
    ```
   `No files found.`
3. ```
     pip install --upgrade galaxy-app@git+https://github.com/galaxyproject/galaxy.git@release_21.09#subdirectory=packages/app
     ```
4. ```
    find . -name "exp_history_to_archive.xml"
    ```
    `./venv/lib/python3.8/site-packages/galaxy/tools/imp_exp/exp_history_to_archive.xml`

**Additional context**
Upgrading pip  from 20.0.2 to 22.1.2 did not help.
Various python versions tried.
First encountered in a github action, and subsequently, locally.

**Comment by mvdbeek** [comment]
We should check this again with the new manifest files

**Comment by nuwang** [comment]
This seems to be resolved in 22.05.
